### PR TITLE
Updated bar graph to be a separate component and use d3 now

### PR DESCRIPTION
**Pull request by saketh-n**

### CODE REVIEW

**Review by devyansh — APPROVED** [review]
Looks good

File: src/charts/barchart.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+    var label = svg.selectAll("text").data(data);
+
+    label
+      .transition()
+      .duration(300)
+      .attr("height", d => yScale(d))
+      .attr("y", d => height - yScale(d) + 15)
+      .text(d => {
+        console.log("hairy balls");
```
Comment (minneker) [inline src/charts/barchart.js]: What is the purpose of this print statement? I don't think we need it lol

File: src/components/main.js [hunk 2]
```diff
@@ -107,14 +62,7 @@ export default function Header() {
...
         </div>
-        {result && (
-          <div className="flex flex-col">
-            <div className="flex items-end">{barGraphResults(result)}</div>
-            <h1 className="mt-3 font-semibold">
-              Seizure Probability at 2 second Intervals
-            </h1>
-          </div>
-        )}
+        {result && <BarChart width={550} height={400} data={result} />}
```
Comment (minneker) [inline src/components/main.js:65]: Nice change, I like how clean the code is when using BarChart.

File: src/charts/barchart.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      .attr("font-family", "sans-serif")
+      .attr("height", 0)
+      .attr("font-size", "14px")
+      .attr("fill", "black")
+      .attr("text-anchor", "middle")
+      .transition()
+      .duration(300)
+      .attr("height", d => yScale(d))
+      .attr("y", d => height - yScale(d) + 15)
+      .attr("transform", "translate(50, -30)");
```
Comment (minneker) [inline src/charts/barchart.js:106]: Should be OK for this PR, but lots of "magic" constants in this file. Maybe we could try and refactor/parameterize in future. For now should be good for purposes of MVP, etc.

**Review by minneker — APPROVED** [review]
lgtm! Really like the BarChart implementation in D3! There is a sus `console.log` statement that I think we need to get rid of, but other than that looks great!

File: src/charts/barchart.js (hunk 1 above)
Comment (saketh-n) [reply src/charts/barchart.js]: Hahaha, good catch Rob. Just some debugging code I forgot to take out.

Awk

File: src/charts/barchart.js (hunk 3 above)
Comment (saketh-n) [reply src/charts/barchart.js:106]: Sure I'll make a note to address that later

**Pull request merged**